src/engine/framework/System.cpp

```diff
@@ -564,7 +564,7 @@ static void Init(int argc, char** argv)
 #endif
 
 	// Print a banner and a copy of the command-line arguments
-	Log::Notice(Q3_VERSION " " PLATFORM_STRING " " ARCH_STRING " " __DATE__);
+	Log::Notice(Q3_VERSION " " PLATFORM_STRING " " STRING(ARCH_STRING) " " __DATE__);
```

Comment:
I guess you need to change tho two instances of STRING(ARCH_STRING) back to XSTRING  (sorry)

Reply:
Done.